qiskit_pkg/setup.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# This file is the setup.py file for the qiskit package. Because python
+# packaging doesn't offer a mechanism to have qiskit supersede qiskit-terra
+# and cleanly upgrade from one to the other, there needs to be a separate
+# package always needs
```

Comment:
I think my commenting on a previous review comment helped this get lost, but this comment's still bust.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/10530/commits/429a7db27f0af7dc2a70336d2492693b0dd83233